### DISCUSSION TITLE
Can't find change to mono command

### DISCUSSION THREAD

**Discussion by tack32** [smplayer-dev/smplayer#861] [General]
I read this was added as a feature a while back but can't find it under "audio". Is it elsewhere or still available? Thanks.

**Comment by Raza-0007** [comment]
I believe it is under `Audio > Stereo mode`

**Reply by tack32** [reply]
I have the latest version but I don't seem to have that. Thanks.
![audio stereo](https://github.com/smplayer-dev/smplayer/assets/93066797/30b7a244-b14e-4930-af1f-032c7809e48e)

**Comment by smplayer-dev** [comment]
Disable the "tablet mode" in the options menu. That hides several options from the menus.

**Reply by tack32** [reply]
That did it! Thanks!
